src/viking.js

```diff
@@ -1,11 +1,107 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength = strength;
+    }
+    attack() {
+        return this.strength;
+
+    }
+    receiveDamage(damage) {
+        
+        this.health -= damage
+
+    }
+
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier{
+    constructor(name, health, strength){
+        super(health, strength)
+        this.name = name
+
+    }
+    receiveDamage(damage) {
+    
+        this.health -= damage;
+    if (this.health > 0) {
+        return `${this.name} has received ${damage} points of damage`
+    } else {
+        return `${this.name} has died in act of combat`
+    }
+  }
+  battleCry() {
+    return `Odin Owns You All!`
+  }
+}
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier {
+    receiveDamage(damage){
+        this.health-=damage;
+        if(this.health > 0) {
+            return `A Saxon has received ${damage} points of damage`
+        } else {
+            return `A Saxon has died in combat`
+        }
+    }
+}
 
 // War
-class War {}
+class War {
+    constructor() {
+    this.vikingArmy = [];
+    this.saxonArmy = []
+    }
+    addViking(Viking) {
+        this.vikingArmy.push(Viking)
+    }
+    addSaxon(Saxon) {
+        this.saxonArmy.push(Saxon)
+    }
+    vikingAttack() {
+        
+    let randSaxon = this.saxonArmy[Math.floor(Math.random()*this.saxonArmy.length)];
+    let randViking = this.vikingArmy[Math.floor(Math.random()*this.vikingArmy.length)];
+    let result = randSaxon.receiveDamage(randViking.strength) 
+    if (randSaxon.health <= 0) {
+        this.saxonArmy.splice(randSaxon, 1)
+    } 
+    return result
+   
+    }
+    saxonAttack() {
+
+       let randSaxon = this.saxonArmy[Math.floor(Math.random()*this.saxonArmy.length)];
+       let randViking = this.vikingArmy[Math.floor(Math.random()*this.vikingArmy.length)];
+       let result = randViking.receiveDamage(randSaxon.strength)
+        if (randViking.health <= 0) {
+            this.vikingArmy.splice(randViking, 1)
+            
+        } 
+        return result
+        
+
+    }
+    
+    showStatus() {
+        if(!this.saxonArmy.length && this.vikingArmy.length) {
+            return `Vikings have won the war of the century!`
+        }
+        
+        else if(!this.vikingArmy.length && this.saxonArmy.length) {
+            return `Saxons have fought for their lives and survived another day...`
+        }
+        else if(this.vikingArmy.length>0 && this.saxonArmy.length > 0) {
```

Comment:
Julia, here, you can get used to the `else`, since you targeted the two other options with the `if` and the `else if`, you can trigger the `else` operator for the remaining answer.